tests/helpers/themed-intl-test-helper.tsx

```diff
@@ -4,7 +4,7 @@
 import React from 'react';
 
 import CompassThemeProvider from 'components/compass_theme_provider/compass_theme_provider';
-import {Theme} from 'mattermost-redux/types/themes';
+import {Theme} from '@mattermost/types/themes';
```

Comment:
These is conversation on how and where to move `theme` https://community.mattermost.com/core/pl/85auue3yytfutkts84rfefynpe